packages/apron/apron.20150820/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "1.2"
+authors: ["Bertrant Jeannet" "Antoine Miné"]
+homepage: "http://apron.cri.ensmp.fr/library/"
+maintainer: "Nicolas Berthier <m@nberth.space>"
+bug-reports: "https://gforge.inria.fr/tracker/?atid=8946&group_id=2625&func=browse"
+license: "LGPL-2.1"
+build: [
+  ["sh" "./configure" "--prefix" "%{apron:share}%"
+  		      "--no-ppl" {! conf-ppl:installed}
+		      "--no-java"
+		      "--absolute-dylibs" { os = "darwin" } ]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "apron"]
+  ["rm" "-r" "-f" "%{apron:share}%"]
+]
+tags: [ "flags:light-uninstall" ]
+depends: [
+  "ocamlfind"
+  "camlidl"
+  "mlgmpidl"
+]
+depopts: [
+  "conf-ppl"
+]
+available: [
+  opam-version >= "1.2" &
+  (os = "linux" | os = "freebsd" | os = "osx" | os = "darwin")
```

Comment:
i would suggest dropping this `os` restriction -- it will probably work on openbsd for example